### DISCUSSION TITLE
Can you port Tab Fern to Firefox?

### DISCUSSION THREAD

**Discussion by justpeanuts** [cxw42/TabFern#269] [Q&A] [answered]
IMO, Firefox lacks an addon like Tab Fern to effectively manage, backup and easily organise and visualise tabs. Can you consider porting the addon to Firefox as well?

**Comment by cxw42** [comment, marked as answer]
#100

**Comment by cxw42** [comment]
Assistance welcome :D .  Thanks for using TF and for letting me know there is a need on FF!
